.travis.yml

```diff
@@ -1,6 +1,12 @@
 language: node_js
-node_js: 7.4.0
+cache:
+  directories:
+    - node_modules
 script:
   - npm install -g knex
   - knex migrate:latest
   - npm test
+after_success:
+  - .travis/on_build_complete.sh "1"
```

Comment:
Also, prefer to avoid another language if possible. Can we do this using a native js/node script?
(Assuming we have a decent amount of Windows devs and/or just devs that don't know bash very well)